aws/cloudformation/bootstrap_standalone.sh.erb

```diff
@@ -110,7 +110,7 @@ rm -f /etc/init.d/{dashboard,pegasus}
 rm -rf $HOME_DIR/apps/build
 rm -rf $HOME_DIR/apps/node_modules
 
-OPTIONS="<%=local_mode ? '-z' : ''%> -b $BRANCH -n $NODE_NAME"
+OPTIONS="<%=local_mode ? '-z' : ''%> -b $BRANCH -n $NODE_NAME -e $ENVIRONMENT"
```

Comment:
Bug: An adhoc not being created in "local mode" was not passing the rack environment to its bootstrap script.

How one might get into this situation:
```sh
rake adhoc:start \
  RACK_ENV=test \ # Not using RACK_ENV=adhoc
  CHEF_SERVER=1 # Attaches to our chef server, not in local mode
```